### PR TITLE
BLOCKS-77 formula operator get replaced with crowdin value [AK]

### PR DESCRIPTION
**Pull request by whati001**
Replace formula operator with crowdin value. -> "MINUS" => "-".

### Your checklist for this pull request
- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Make sure you use BLOCKS instead of CATROID in your commit message
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed (Actions)
- [x] Post a message in the *#catblocks* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by wslany** [comment]
Note that https://github.com/Catrobat/Catblocks/pull/97 seems to solve _some_ of the issues. There, https://share.catrob.at/pocketcode/project/1401043a-72ef-11ea-a2ae-000c292a0f49 looks as follows:
image

File: src/js/catblocks_msgs.js [hunk 1]
```diff
@@ -237,6 +246,20 @@ Blockly.CatblocksMsgs.locales["af"] = {
   "STITCH_ZIGZAG": "undefined %1 undefined %2",
```
[the remark this answers is not in the document]
Comment (whati001) [reply src/js/catblocks_msgs.js]: Will update the language generation to use en_GB as default. Currently I/we assumed that each catroid language file includes all values.

File: src/js/catblocks_msgs.js [hunk 2]
```diff
@@ -237,6 +246,20 @@ Blockly.CatblocksMsgs.locales["af"] = {
...
   "STITCH_TRIPLE": "undefined %1",
   "STITCH_STOP": "undefined",
+  "NOTEQUAL": "≠",
+  "LESSERTHAN": "<",
+  "GREATERTHAN": ">",
+  "AND": "and",
+  "OR": "or",
+  "NOT": "not",
+  "GREATEREQUAL": "≥",
+  "LESEREQUAL": "≤",
```
[the remark this answers is not in the document]
Comment (whati001) [reply src/js/catblocks_msgs.js]: The reason for the '0.1' value inside the WaitBrick is due to wrong field naming inside Catblocks code. This was fixed with #97, as you mentioned below.

File: src/js/parser/parser.js [hunk 3]
```diff
@@ -260,6 +264,22 @@ function parseBrick(brick) {
   return currentBrick;
 }
 
+/**
+ * Return crowding value or default
```
[the remark this answers is not in the document]
Comment (whati001) [reply src/js/parser/parser.js]: Fixed spelling issue.

File: test/jsunit/parser/parser.test.js [hunk 4]
```diff
@@ -331,5 +331,20 @@ describe('Catroid to Catblocks parser tests', () => {
         return (testValue !== undefined && testValue.innerHTML.length === 0);
       })).toBeTruthy();
     });
+
+    /** 
+     * Test if parser handles formular operator properly
+     */
+    test('Test if parser handels formular operator properly', async () => {
```
[the remark this answers is not in the document]
Comment (whati001) [reply test/jsunit/parser/parser.test.js]: Will have a look at it if there is some time left.
Removed function decoration from all tests.

File: src/js/catblocks_msgs.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (wslany) [reply src/js/catblocks_msgs.js]: Ahh, the default is US English = en (without any regional variant), not en_GB. 

I believe there should be a setting in CrowdIn when exporting the data that automatically fills the empty translations with the default en strings, AFAIK.

**Pull request merged**